_config.yml

```diff
@@ -24,3 +24,11 @@ github:
 # Your search_site_handle value is from the search.gov settings page.
 
 # search_site_handle: searchgov-site-handle
+
+# Change the theme variant. Current options are: "beach"
```

Comment:
Is this leftover from a copy-pasta?

Reply:
leftover from removing some functionality, good catch 🎣